OpenEdXMobile/res/layout/collapsible_toolbar_layout.xml

```diff
@@ -12,25 +13,31 @@
         android:id="@+id/collapsingToolbar"
         android:layout_width="match_parent"
         android:layout_height="@dimen/edx_quadruple_margin"
-        app:collapsedTitleGravity="center_vertical"
-        app:collapsedTitleTextAppearance="@style/MainDashboardCollapsedToolbarTitle"
-        app:expandedTitleGravity="center_vertical"
-        app:expandedTitleMargin="0dp"
         app:expandedTitleMarginStart="@dimen/edx_default_margin"
-        app:expandedTitleTextAppearance="@style/MainDashboardExpandedToolbarTitle"
         app:layout_scrollFlags="scroll|snap|exitUntilCollapsed">
 
         <androidx.appcompat.widget.Toolbar
             android:layout_width="match_parent"
             android:layout_height="@dimen/collapsed_toolbar_height"
+            android:layout_gravity="center_vertical"
             android:background="@color/neutralWhite"
             android:gravity="center_vertical"
             android:paddingStart="@dimen/container_padding"
             android:paddingEnd="@dimen/edx_default_margin"
-            app:contentInsetLeft="0dp"
             app:contentInsetStart="0dp"
-            app:layout_collapseMode="pin"
-            app:titleTextAppearance="@style/MainDashboardCollapsedToolbarTitle" />
+            app:layout_collapseMode="parallax">
+
+            <com.google.android.material.textview.MaterialTextView
+                android:id="@+id/tv_title"
+                style="@style/MainDashboardToolbarTitle"
+                android:layout_width="wrap_content"
```

Comment:
Should be set to `match_parent` to support RTL.

Reply:
for RTL, the view moved to the right side, so no need. but adding for the safe side.